dht/sharder.hh

```diff
@@ -116,8 +116,6 @@ class selective_token_range_sharder {
     dht::token _start_token;
     std::optional<range_bound<dht::token>> _start_boundary;
```

Comment:
changelog

Reply:
The patch changes repair/ but the subject says selective_token_range_sharder
```
commit 209f912f954bd632a330e3372e58a80a6e3bc083
Author: Piotr Jastrzebski <piotr@scylladb.com>
Date:   Thu Dec 19 19:08:30 2019 +0100

    selective_token_range_sharder: stop calling global_partitioner()
    
    Signed-off-by: Piotr Jastrzebski <piotr@scylladb.com>
---
 repair/repair.hh |  2 ++
 repair/repair.cc | 30 ++++++++++++++++++++++--------
 2 files changed, 24 insertions(+), 8 deletions(-)
```

Reply:
I hate github and its way of showing comments on wrong commits!!!

Reply:
I added a commit message to explain the changes in repair.